src/NUnitFramework/framework/Internal/TestFilter.cs

```diff
@@ -94,7 +94,7 @@ public virtual bool IsExplicitMatch(ITest test)
         /// </summary>
         /// <param name="test">The test to which the filter is applied</param>
         /// <returns>True if the filter matches the an ancestor of the test</returns>
-        private bool MatchParent(ITest test)
+        public bool MatchParent(ITest test)
```

Comment:
Yes, but that has nothing to do with BaseFilter, which is only a TestFilter.

 

From: CharliePoole [mailto:notifications@github.com] 
Sent: Saturday, October 03, 2015 11:55 PM
To: nunit/nunit
Cc: Neil Colvin
Subject: Re: [nunit] Fix NotFilter so -exclude works correctly (#862)

 

In src/NUnitFramework/framework/Internal/TestFilter.cs <https://github.com/nunit/nunit/pull/862#discussion_r41094222> :

> @@ -94,7 +94,7 @@ public virtual bool IsExplicitMatch(ITest test)
>          /// </summary>
>          /// <param name="test">The test to which the filter is applied</param>
>          /// <returns>True if the filter matches the an ancestor of the test</returns>
> -        private bool MatchParent(ITest test)
> +        public bool MatchParent(ITest test)

But NotFIlter inherits from TestFilter.

—
Reply to this email directly or view it on GitHub <https://github.com/nunit/nunit/pull/862/files#r41094222> .  <https://github.com/notifications/beacon/AFh64S2StCxGxckyRtLV3Dd5hXeRIag-ks5o4JqhgaJpZM4GIcgJ.gif>